src/plugins/intel_gpu/tests/unit/test_cases/loop_gpu_test.cpp

```diff
@@ -432,10 +432,18 @@ TEST(loop_gpu, basic_concat_nested_cached) {
     test_loop_gpu_basic_concat_nested<float>(true);
 }
 
-static void test_loop_gpu_wo_trip_count(ov::PartialShape body_input_layout, bool is_caching_test = false) {
+
+
+static void test_loop_gpu_wo_trip_count(ov::PartialShape body_input_layout,
+                                        ov::PartialShape sliced_layout,
+                                        std::vector<float> input_data,
+                                        std::vector<float> expected_output_data,
+                                        size_t axis,
+                                        size_t exit_value,
+                                        bool is_caching_test = false) {
     auto& engine = get_test_engine();
 
-    auto e_input_layout = cldnn::layout{ { 1, 1, 5, 4 }, data_types::f32, format::bfyx };
+    auto e_input_layout = cldnn::layout{ sliced_layout, data_types::f32, format::bfyx };
```

Comment:
why is it `sliced_layout`? As far as I understood, it is the layout before slice. Better name would be something like `original_layout` or `whole_layout`..

Reply:
Modified.